packages/fetchai/connections/ledger/contract_dispatcher.py

```diff
@@ -356,7 +356,7 @@ def _validate_and_call_callable(
         :return: the data generated by the method.
         """
         try:
-            method_to_call = getattr(contract, message.callable)
+            method_to_call = getattr(contract, "default_method_call")
```

Comment:
We will need to make some checks or adjust the existing ones in the lines before, but is this what we are looking for? We always call the default method that is resolved on the contract base, or do we only do this if we do not finde the requested method?